SmartDeviceLink/public/SDLBodyInformation.m

```diff
@@ -17,23 +17,21 @@ @implementation SDLBodyInformation
 
 - (instancetype)initWithParkBrakeActive:(BOOL)parkBrakeActive ignitionStableStatus:(SDLIgnitionStableStatus)ignitionStableStatus ignitionStatus:(SDLIgnitionStatus)ignitionStatus {
     self = [self init];
-    if (!self) {
-        return nil;
+    if (self) {
+        self.parkBrakeActive = @(parkBrakeActive);
+        self.ignitionStableStatus = ignitionStableStatus;
+        self.ignitionStatus = ignitionStatus;
```

Comment:
Can you change this back to the way the generator created it? Then the formatting will be consistent with most of the other classes in the project.